src/components/Header.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+interface Props {
+  onAddTodo: (e: React.FormEvent) => void;
+  isAllCompleted: boolean;
+  handleInputChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  todosLength: number;
+}
+
+const Header: React.FC<Props> = ({
+  onAddTodo,
+  isAllCompleted,
+  handleInputChange,
+  todosLength,
+}) => {
+  const handleClearInput = () => {
+    handleInputChange({
+      target: { value: '' },
+    } as React.ChangeEvent<HTMLInputElement>);
+  };
```

Comment:
I don't get what is the reason to pass an empty string as `target.value`, you need to change types to avoid it or this function do something else instead of clearing input